draft-ietf-jsonpath-base.md

```diff
@@ -646,7 +649,8 @@ not encode Unicode characters.
 #### Syntax
 {: unnumbered}
 
-The dot wild card selector has the form `.*`.
+The dot wild card selector has the form `.*` as defined in the
+following syntax:
```

Comment:
This is getting a little wordy and would probably need to be applied throughout the document for consistency. Isn't it obvious that a piece of ABNF in a syntax section is the normative syntax for the construct being discussed?

Reply:
The reason for the wordiness is that the sentence ended in ".*." before -- at least in the plaintext rendering.  So we now have to contort our sentences to avoid that run-in.